ambry-test-utils/src/main/java/com/github/ambry/utils/TestUtils.java

```diff
@@ -283,6 +283,8 @@ static class ZkServerWrapper {
     public ZkServerWrapper(String dataDir, String logDir, int port) {
       this.dataDir = new File(dataDir);
       this.dataLogDir = new File(logDir);
+      this.dataDir.delete();
```

Comment:
Same here, why you need to add this delete(), just trying to understand your intension, thx!

Reply:
This is just to ensure that the ZK log folder from previous run is cleaned up and we start from an empty dir